### ISSUE TITLE
cinelerra-gg: program uses default.profile

### ISSUE THREAD

**Issue by anomalocaris452** [netblue30/firejail#5533]
Environment:

* cinelerra-gg from chaotic-aur / aur (Arch Linux)
* firejail 0.9.70

Log:

```
Reading profile /etc/firejail/default.profile
Reading profile /etc/firejail/disable-common.inc
Reading profile /etc/firejail/disable-programs.inc

** Note: you can use --noprofile to disable default.profile **

Parent pid 131431, child pid 131432
Warning: /sbin directory link was not blacklisted
Warning: /usr/sbin directory link was not blacklisted
Child process initialized in 206.13 ms
Cinelerra Infinity - built: Dec  1 2022 03:55:24
git://git.cinelerra-gg.org/goodguy/cinelerra.git
(c) 2006-2019 Heroine Virtual Ltd. by Adam Williams
2007-2020 mods for Cinelerra-GG by W.P.Morrow aka goodguy
Cinelerra is free software, covered by the GNU General Public License,
and you are welcome to change it and/or distribute copies of it under
certain conditions. There is absolutely no warranty for Cinelerra.

lang changed from '' to 'en_US'
MWindow::init_plugins: can't open plugin index: /home/senpai/.bcast5/Cinelerra_plugins
MWindow::init_ladspa_plugins: can't open ladspa plugin index: /home/senpai/.bcast5/ladspa_plugins._usr_lib_cinelerra-gg_ladspa
MWindow::init_theme: prefered theme Cakewalk not found.
MWindow::init_theme: theme_plugin not found.
unjoined tids / owner 1
  00007fdf5aab66c0 / 00007fdf734b4800 12BC_Clipboard
```

It works with the `--noprofile` + `--net=none` flags.

**Comment by rusty-snake** [comment]
Firejail 0.9.70 does not have a profile for cinelerra-gg.

**Comment by rusty-snake** [comment]
@glitsj16 The profile on master misses the `.profile`.

https://github.com/netblue30/firejail/blob/master/etc/profile-a-l/cinelerra-gg

**Comment by glitsj16** [comment]
@rusty-snake Fixed! With all the CI stuff we have it's weird this mistake of mine got through unnoticed. Anyway, very well spotted, thank you.

**Comment by rusty-snake** [comment]
We could add a check for it because it is not that uncommon:

```console
$ bash -O extglob -O nullglob -c 'echo "Wrong named files:" etc/profile-{a-l,m-z}/!(*.profile)'
Wrong named files: etc/profile-m-z/xlinks2
```

**Comment by kmk3** [comment]
Fixed by commit 52a13f70c ("Rename cinelerra-gg to cinelerra-gg.profile",
2022-12-19) on firejail 0.9.72.
